unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/SettingsPanelHUDView.cs

```diff
@@ -26,6 +37,11 @@ public class SettingsPanelHUDView : MonoBehaviour
         private IHUD hudController;
         private ISettingsPanelHUDController settingsPanelController;
 
+        private void Awake()
+        {
+            openAction.OnTriggered += OpenAction_OnTriggered;
```

Comment:
Having this suscription here and all the rest under `Create()` looks off. There's a necessity to split the suscription responsibilities here?

Reply:
You're right, I can move the subscription to the Initialize method and remove the Awake. Done in the feedback PR: https://github.com/decentraland/explorer/pull/1784 Thanks!